portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,36 @@
 #content {
   margin-left: auto;
   margin-right: auto;
+  padding-top: 75px;
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.icon-bar a {
+  padding: 10px;
+  color: gray;
 }
+
+.icon-bar a:hover {
+  color: black;
+}
+
+.navbar ul {
+  position: fixed;
+  padding-left: 25px;
+  padding-top: 25px;
+}
+
+.navbar .icon-bar {
+  position: fixed;
+  right: 0;
+  padding-top: 25px;
+}
+
+.bg-light {
+  padding-top: 50px;
+  padding-bottom: 50px;
+  position: fixed;
+  left: 0;
+  width: 100%;
+}
```

Comment:
super nit: lets have a consistent order of css properties for all blocks - so lets say you have position listed before padding-<top/bottom/left/right> in all of the blocks (and same for other things you add)